v2/tests/log_context_property_basic_types_ut/log_context_property_basic_types_ut.c

```diff
@@ -178,17 +178,17 @@ static void int32_t_to_string_with_truncation_succeeds(void)
 
 /* Tests_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_002: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).to_string shall call snprintf with buffer, buffer_length and format string PRId64 and pass in the values list the int64_t value pointed to be property_value. ]*/
 /* Tests_SRS_LOG_CONTEXT_PROPERTY_BASIC_TYPES_01_003: [ LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).to_string shall succeed and return the result of snprintf. ]*/
-static void int32_t_to_string_with_negative_value_succeeds(void)
+static void int64_t_to_string_with_negative_value_succeeds(void)
 {
     // arrange
     char buffer[TEST_BUFFER_SIZE];
-    int32_t int32_t_value = -1;
+    int64_t int64_t_value = -1;
 
     setup_mocks();
     setup_expected_snprintf_call();
 
     // act
-    int result = LOG_CONTEXT_PROPERTY_TYPE_IMPL(int32_t).to_string(&int32_t_value, buffer, TEST_BUFFER_SIZE);
+    int result = LOG_CONTEXT_PROPERTY_TYPE_IMPL(int64_t).to_string(&int64_t_value, buffer, TEST_BUFFER_SIZE);
 
     // assert
     POOR_MANS_ASSERT(result > 0);
```

Comment:
>result > 0 [](http://example.com/codeflow?start=21&length=10)

There can be a better precision regarding the number of bytes, in this case it is "2".

Reply:
Fixed.